modules/openshift-cluster-maximums.adoc

```diff
@@ -14,14 +14,14 @@
 | 2,000
 | 250
 | 500
-| 2,000
+| 500
 
 | Number of pods ^[1]^
 | 150,000
 | 150,000
 | 62,500
 | 62,500
-| 150,000
+| 62,500
```

Comment:
@chaitanyaenr In the companion PR to this work, the value was 62,000 and not 62,500. Is that intentional?
Also, what is the motivation to have this PR against enterprise-4.6 only? Should these changes apply to master so that the changes can be cherrypicked forward into 4.6 (and 4.7 so that we can add 4.7 metrics later)?

Reply:
@ahardin-rh the number of pods should be 62,500, it was a typo in the companion PR. The environment module ( openshift-cluster-maximums-environment.adoc ) is included in the planning-your-environment-according-to-object-maximums.adoc page of the master branch but is missing in caseof enterprise-4.6 branch.